Makefile

```diff
@@ -302,17 +304,17 @@ shared-varlibcontainers:
 vm-backup:
 	scp $(SSH_FLAGS) recert_script.sh core@$(VM_NAME):/var/tmp
 	ssh $(SSH_FLAGS) core@$(VM_NAME) sudo /var/tmp/recert_script.sh backup
-	virsh shutdown $(VM_NAME)
-	@until virsh domstate $(VM_NAME) | grep -qx 'shut off' ; do echo -n . ; sleep 5; done; echo
-	cp "$(LIBVIRT_IMAGE_PATH)/$(VM_NAME).qcow2" "$(LIBVIRT_IMAGE_PATH)/$(VM_NAME)-$(VERSION)-backup.qcow2"
-	virsh start $(VM_NAME)
+	virsh --connect=$(VIRSH_CONNECT) shutdown $(VM_NAME)
```

Comment:
Create a `virsh` variable with the command and parameters, so every time we use virsh we just write `$(virsh)` instead

Same as we're doing with `oc`